src/entity/Notification.ts

```diff
@@ -32,6 +32,7 @@ export type NotificationType =
   | 'squad_new_comment'
   | 'squad_reply'
   | 'squad_post_viewed'
+  | 'squad_subscribe_to_notification'
```

Comment:
What about something like `squad_first_post_notification`?
That seems more explanatory of the notification

Reply:
I was thinking more about the intent. But I'm open to whichever fits. Would love to hear what others think too.

Reply:
`squad_subscribe_to_notification` feels like it matches more with the rest of the values in `NotificationType`.. but we can always flip a coin to decide 😆